.github/dependabot.yml

```diff
@@ -14,6 +14,16 @@ updates:
     allow:
       - dependency-name: "bpmn-js"
         dependency-type: "all"
+      - dependency-name: "diagram-js"
```

Comment:
Should we add `bpmn-io/element-template-chooser` and `bpmn-io/element-template-icons-renderer` as well?

Reply:
Added via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/746/commits/2ced6a86ed9e216b51d7e4617b75a528220c9381.